src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -154,6 +154,18 @@ private function addFormSection(ArrayNodeDefinition $rootNode)
                                 ->scalarNode('field_name')->defaultValue('_token')->end()
                             ->end()
                         ->end()
+                        // This node can be set to false by default from Symfony 5.0
+                        ->booleanNode('legacy_error_messages')
+                            ->defaultTrue()
+                            ->validate()
+                                ->ifTrue()
+                                ->then(function ($v) {
+                                    @trigger_error('Setting the "framework.form.legacy_error_messages" to "true" is deprecated and the option will have no effect as of Symfony 5.0.', E_USER_DEPRECATED);
```

Comment:
We should already add a (commented) exception that will be thrown as of 5.0.

Reply:
I currently don't have time to finish this. Could you work on it?

Reply:
Thanks for the feedback. I try to finish it.